.lighthouserc.js

```diff
@@ -0,0 +1,234 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS-IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Config for lighthouse-ci
+ */
+
+ // To run Lighthouse Checks type lhci autorun
+ module.exports = {
+    ci: {
+      collect: {
+        numberOfRuns: 1,
+        startServerCommand: 'python -m scripts.start',
```

Comment:
Is this needed? We run the server in the Python anyway?

Reply:
I did it because I was facing a similar issue to https://github.com/oppia/oppia/pull/9295#discussion_r424561360 
and decided to move it to the config. I think I can move it back by using the fix I did
in `run_lighthouse_checks()`

Reply:
So can you remove it then?

Reply:
I actually think it's better to run the `python -m scripts. start` it in the config. Running it in the `run_lighthouse_tests` script generates a lot of unnecessary logs and also forces me to terminate the server afterward as well.